src/core/renderers/webgl/managers/FilterManager.js

```diff
@@ -327,7 +327,7 @@ export default class FilterManager extends WebGLManager
         // TODO Cacheing layer..
         for (const i in uniformData)
         {
-            const type = uniformData[i].type.toLowerCase();
+            const type = uniformData[i].type;
```

Comment:
`const {type} = uniformData[i];`?

Reply:
Seems like pointless change to me.

Reply:
Only a suggestion. less code

Reply:
ok, next time when i see multiple of these :)